extensions/reviewed/PlayerAvatar.json

```
@@ -0,0 +1,535 @@
+                {
+                  "type": {
+                    "value": "StringVariable"
+                  },
+                  "parameters": [
+                    "Value",
+                    "!=",
+                    "UserID"
+                  ]
+                }
+              ],
+              "actions": [
+                {
+                  "type": {
+                    "value": "SetStringVariable"
+                  },
+                  "parameters": [
+                    "UserID",
+                    "=",
+                    "Value"
+                  ]
+                }
+              ],
+              "events": [
+                {
+                  "type": "BuiltinCommonInstructions::JsCode",
+                  "inlineCode": [
+                    "// Check if there is no multiplayer in game, exit the function.",
+                    "if (!gdjs.multiplayerMessageManager) return;",
+                    "",
+                    "/** @type {string} */",
+                    "const parameterUserID = eventsFunctionContext.getArgument(\"Value\");",
+                    "",
+                    "// If the userID isn't valid, exit the function.",
+                    "if (parameterUserID.length < 20 ||",
+                    "    parameterUserID.length > 36) return;",
+                    "",
+                    "// Get the PixiJS renderer of the Avatar Sprite",
+                    "const spriteRenderer = objects[0].getRendererObject();",
+                    "",
+                    "// Build the URL to request to the API the profile from a GDevelop userID.",
+                    "const url = \"https://api.gdevelop.io/user/user-public-profile/\" + parameterUserID;",
+                    "",
+                    "// Get the reponse from the API when possible then change the texture by the image from the GDevelop user profile.",
+                    "gdjs._extensionAvatar.getGDevelopProfile(url).then((json) => {",
+                    "  const baseTexture = PIXI.BaseTexture.from(json.iconUrl.split(\"?\")[0] + \"?s=200\");",
+                    "  spriteRenderer.texture = new PIXI.Texture(baseTexture);",
+                    "});",
+                    ""
+                  ],
+                  "parameterObjects": "Avatar",
+                  "useStrict": true,
+                  "eventsSheetExpanded": true
+                }
+              ]
+            }
+          ],
+          "parameters": [
+            {
+              "description": "Object",
+              "name": "Object",
+              "supplementaryInformation": "PlayerAvatar::PlayerAvatar",
+              "type": "object"
+            }
+          ],
+          "objectGroups": []
+        }
+      ],
+      "propertyDescriptors": [
+        {
+          "value": "0",
+          "type": "Number",
+          "unit": "Dimensionless",
+          "label": "Lobby player number",
+          "description": "Leave it to 0 to show the current player.",
```

I guess we should revert this description too.
```suggestion
          "description": "",
```
What about the default value? Is `0` ok?